python/lsst/afw/image/visitInfo.cc

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
Yes to both questions. What happened is that my metadata ticket was merged after you started the pybind11 work, so some new header files were not picked up.